LibDeflate.lua

```diff
@@ -1881,7 +2011,12 @@ local function Deflate(configs, WriteBits, WriteString, FlushWriter, str
 					end
```

Comment:
I probably should check "newSize" here
`
if newSize > 0 then
    hash_tables[k] = new
else
    hash_tables[k] = {}
end
`

Reply:
Also the code style needs to be fixed here, I use "under_score" for variables. CamelCase for functions.